countryhubs/reports/blueprints/2024-11-23-Greece-blueprint-chws

```diff
@@ -0,0 +1,31 @@
+## Template 2407: Blueprint support through Country Hubs Workshop (CHWS) Reporting 
+
+### Summary
+
+1. **Country of CHWS**:  
+   Greece  
+
+2. **Audience Familiarity with Blockchain and Crypto**:  
+   Begginer-Intermediate  
+
+3. **Participant Count**:  
+   3 
+
+4. **Founders**:  
+   No founders,all of them seemed interested on how Cardano could be used to bootstrap a foudning journey. 
+
+5. **Business Topics Covered**:  
+   - Cardano Constitution: Its current phase, completion percentage, voting process, and the role of DReps.  
+   - Custody vs. Non-custody wallets: Key differences and functionality, UTxO model, popular wallets in the Cardano ecosystem, and wallet-staking pool connections.  
+   - Setting up a staking pool: Requirements, pledge impact, operator responsibilities, fees, and epoch rewards.  
```

Comment:
Hi Giorgos: Could you elaborate more on the reuslts these two points: wallets and stakepool.
The use-cases when and how to use a stakepool in case of a business context. 
How to connect and support other projects with your stakepool.
Also, is it worth it from a business perspective?

Wallets are also a great topic, when it comes to how to introduce wallets to non crypto natives. 
What were the key differences between the wallets. 

This is all great content if turned into text.